src/System/DolDoc/DocPutKey.ZC

```diff
@@ -447,9 +447,21 @@ public U0 DocPutKey(CDoc *doc, I64 ch=0, I64 sc=0)
 
 									case SC_F6:
 										if (sc & SCF_KEY_DESC)
-											KeyDescSet("Cmd /God Song");
+										{
+											if (sc & SCF_SHIFT)
```

Comment:
@Midnoclose 
same as TempleOS https://github.com/cia-foundation/TempleOS/blob/c26482bb6ad3f80106d28504ec5db3c6a360732c/Adam/DolDoc/DocPutKey.HC#L326-L341

Reply:
It's fine if you want to change it. I'm just saying the only reason I did it the way I did was because it was from TempleOS. I'm not sure if I even have the ability to approve the changes you're suggesting but since you made them and since I'm not against them, please add them so you can mark this PR as approved. All I care about is getting the Doodle function back into Zeal.

Reply:
@Midnoclose i dont mind changing it, but it should be throughout the whole file or the entire related feature not just what he's adding, which kinda falls out of scope from his "Adding GodDoodle"  PR